docs/content/contribute/define-meeting/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OSCAL DEFINE Meeting
+description: Details about the OSCAL DEFINE Meeting
+layout: devlunch
+aliases:
+- /contribute/model-review/
+toc:
+  headingselectors: "h3,h4"
+  includeHtml: true
+weight: 20
+---
+
+{{% usa-intro %}}OSCAL DEFINE (**D**evelop **E**nhancements, **F**uture **I**mplementations and **N**ew **E**ducation) is a new meeting to discuss the research and educational pursuits of OSCAL using an iterative and collaborative approach with the community.  Our goal is to establish an OSCAL research framework that allows topics to receive the necessary definition and understanding sufficient to create educational content, demonstrate usage in practice, gain community’s endorsement, and serve as a catalyst for continued development of OSCAL.{{% /usa-intro %}}
+
+{{% usa-tag %}}Day and Time{{% /usa-tag %}} Monthly Thursday at 11:00 AM ET, starting February 16, 2023
```

Comment:
I suggest: "Monthly, third Thursday of the month at 11:00 AM ET, starting February 16, 2023". Saying only Monthly on Th starting... is not clear which Th out of 4 a month will have.

Reply:
```suggestion
{{% usa-tag %}}Day and Time{{% /usa-tag %}} Monthly, third Thursday of the month at 11:00 AM ET, starting February 16, 2023
```

Reply:
That's a good catch, I actually meant to clean that up when updating the date and rebasing earlier today and I clearly got distracted. Fixed.